### DISCUSSION TITLE
Add exceptions for months

### DISCUSSION THREAD

**Discussion by luisrodx** [iSoron/uhabits#1130] [Feature Requests] [answered]
Hello all,

It should be accepted to perform a habit with "x exceptions per month". 
For example, if you want to allow one exception per month, you can't simulate that on the app because different months have different number of days. You can only select something like "29 or 30 times per month", meaning that the former will be incorrect since it allows 2 exceptions on months with 31 days, and the latter will be incorrect because it doesn't allow for exceptions on months with 30 days.

Maybe this might seem a bit pedantic, but I am sure I had this working alright with something like "29 out of 30 days" (close enough for me) but I think that option must have been lost with a certain update.

Thank you!

**Comment by hiqua** [comment, marked as answer]
> I am sure I had this working alright with something like "29 out of 30 days" (close enough for me) but I think that option must have been lost with a certain update.

This is coming back: https://github.com/iSoron/uhabits/pull/1079

So I guess when this is released, you'll be fine with it, right?

**Reply by luisrodx** [reply]
Yes, I would fine with that option too. Thanks!
